kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/asset/provider/BaseAsset.java

```diff
@@ -399,9 +365,8 @@ protected String getFactoryPid() {
      */
     private boolean isDriverAttrbuteDefinition(final String oldAdId) {
         requireNonNull(oldAdId, message.oldAdNonNull());
-        return (oldAdId != ASSET_DESC_PROP.value()) && (oldAdId != ASSET_DRIVER_PROP.value())
-                && (oldAdId != NAME.value()) && (oldAdId != TYPE.value()) && (oldAdId != VALUE_TYPE.value())
-                && (oldAdId != SEVERITY_LEVEL.value());
+        return oldAdId != ASSET_DESC_PROP.value() && oldAdId != ASSET_DRIVER_PROP.value() && oldAdId != NAME.value()
```

Comment:
It's not in what we decided for our cleanup profile. Furthermore usually also Sonar complains for those brackets.